structures/v3/structures.go

```diff
@@ -64,32 +59,3 @@ var (
 )
 
 type ObjectID = primitive.ObjectID
-
-var (
-	DefaultRoles     []*Role = nil
-	defaultRolesLock sync.Mutex
-)
-
-func FetchDefaultRoles(ctx context.Context, mngo mongo.Instance) []*Role {
```

Comment:
This is removed, but do we have a better idea for implementing default permissions?

Reply:
Uh, we should likely cache it in redis.

Reply:
Right yeah, that's pretty true
yeah that's true